src/test/testType.ts

```diff
@@ -125,9 +125,15 @@ export class TestTypeImpl {
   }
 
   private _setTimeout(timeout: number) {
+    const suite = currentlyLoadingFileSuite();
```

Comment:
currentlyLoadingFileSuite returns the active describe suite? Strange

Reply:
Yeah, we'll refactor it.